Testscripts/Windows/KDUMP-TestScript.ps1

```diff
@@ -135,15 +138,18 @@ function Main {
     Write-LogInfo "Trigger the kernel panic..."
     if ($nmi -eq 1) {
         # Waiting to kdump_execute.sh to finish execution.
+        Write-LogInfo "Let KDUMP-Execute.sh complete."
```

Comment:
I see that the end of other sentence doesn't have ".", so I think if we could keep accordance is much better.

Reply:
Fixed in a97a74a326d176abd7507feb4aa7a80d179bad90